.changelog/10401.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+return an invalid record when asked for an addr dns with type other then A and AAAA.
```

Comment:
```suggestion
dns: return an invalid record when asked for an addr dns with type other then A and AAAA.
```

We generally prefix the message with a label of some kind. I don't think we have a good convention for how to choose that label yet, but `dns` is probably a same option